test/ui-testing/pick-lists.js

```diff
@@ -0,0 +1,268 @@
+          .wait('#clickable-save-pick-lists-0')
+          .click('#clickable-save-pick-lists-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find edited pick list ${editedPickList}`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .wait('#editList-pick-lists')
+          .evaluate(list => {
+            const listRows = [...document.querySelectorAll('#editList-pick-lists')].map(e => e.textContent);
+            const listElement = listRows.find(r => r.indexOf(list) >= 0);
+            if (!listElement) {
+              throw Error(`Could not find row with the edited list ${list} but found ${listElement}`);
+            }
+          }, editedPickList)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should not find pick list ${pickList}`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-lists"]')
+          .click('a[href="/settings/licenses/pick-lists"]')
+          .wait('#editList-pick-lists')
+          .evaluate(list => {
+            const row = document.evaluate(
+              `//*[@id="editList-pick-lists"]//div[.="${list}"]`,
+              document,
+              null,
+              XPathResult.FIRST_ORDERED_NODE_TYPE
+            ).singleNodeValue;
+            if (row != null) {
+              throw Error(`Should not find row with list ${list}`);
+            }
+          }, pickList)
+          .then(done)
+          .catch(done);
+      });
+
+
+      it(`should create pick list value ${pickListValue} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-list-values"]')
+          .click('a[href="/settings/licenses/pick-list-values"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#categorySelect')
+          .type('#categorySelect', editedPickList)
+          .wait('#clickable-add-pick-list-values')
+          .click('#clickable-add-pick-list-values')
+          .wait('input[name="items[0].label"]')
+          .type('input[name="items[0].label"]', pickListValue)
+          .wait('#clickable-save-pick-list-values-0')
+          .click('#clickable-save-pick-list-values-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find pick list value ${pickListValue} in list`, done => {
+        nightmare
+          .wait('#editList-pick-list-values')
+          .evaluate(value => {
+            const valueRows = [...document.querySelectorAll('#editList-pick-list-values')].map(e => e.textContent);
```

Comment:
I don't think this `evaluate` is working as intended. The `querySelectorAll` is just going to return the single element for the item by the ID `editList-pick-list-values`.

You could try `[...document.querySelectorAll('#editList-pick-list-values [role="row"]')]` to fetch the _rows_ rather than the list itself.